### PR TITLE
merged it with the file form CapitalAsterisk

### PR DESCRIPTION
**Pull request by haennes**

### CODE REVIEW

**Comment by EhWhoAmI** [comment]
I'll assume that the edits in `README.md` and `docs/architecture.md` are done by Captain Asterisk and `docs/building.md` is done by you. 

First, you don't need the greater than signs, usually you would use them when you are making a quote, which you are not doing here. 

I'm still going over the pull request, but it looks good so far.

**Comment by haennes** [comment]
> I'll assume that the edits in `README.md` and `docs/architecture.md` are done by Captain Asterisk and `docs/building.md` is done by you. 
> 
> First, you don't need the greater than signs, usually you would use them when you are making a quote, which you are not doing here. 
> 
> I'm still going over the pull request, but it looks good so far. 

You are right I haven't touched the readme.md at all

I was trying to make an indent tbh.
How would I accomplish that ?

**Comment by EhWhoAmI** [comment]
There is no way to indent in markdown, but I don't think we need to indent. Indenting will probably only make it messy anyway.

**Comment by z-adams** [comment]
I agree, get rid of the indentation, it just makes the file look noisy.

Some of this information is also out-of-date, do you want me to just commit some fixes directly to your PR?

**Comment by haennes** [comment]
> I agree, get rid of the indentation, it just makes the file look noisy.
> 
> Some of this information is also out-of-date, do you want me to just commit some fixes directly to your PR?

Sure
Let me fix the indentation real quick

**Comment by haennes** [comment]
Done @z-adams you can now "force push " and change the outdated things

**Comment by EhWhoAmI** [comment]
Looks good to me

**Review by z-adams — CHANGES_REQUESTED** [review]
Don't merge anything until we figure out a way for me to include my fixes (see discord chat). Also, I'm going to add @Capital-Asterisk to the reviewers to make sure he's okay with his branch being merged in this way.

**Review by Capital-Asterisk — APPROVED** [review]
It looks all good, not much has really changed compared to my branch. I was thinking of adding a bit more, but there isn't really a point in pushing all the documentation in one big PR.

File: docs/architecture.md
Comment (jonesmz) [inline docs/architecture.md:39]: Seems to be an extra space or tab char here

File: docs/building.md
Comment (jonesmz) [inline docs/building.md:28]: Recommend adding --depth=10 and --submodule-depth=10 (make sure I'm remembering the names of these correctly !!)

File: docs/building.md
Comment (jonesmz) [inline docs/building.md:36]: git submodule update --init --recursive

File: docs/building.md
Comment (jonesmz) [inline docs/building.md:41]: What? Why? I've never seen this happen.

File: docs/building.md
Comment (jonesmz) [inline docs/building.md:47]: Link to github actions config for Linux and windows as examples of how to do this as well.

**Review by jonesmz — APPROVED** [review]
I approve of these changes, but did add several comments. I would appreciate if my comments were addressed before merging. But won't be upset if it gets merged as is.

File: docs/building.md
Comment (Capital-Asterisk) [reply docs/building.md:41]: This was a really old issue that doesn't happen anymore. This was copy-pasted straight from the wiki.

**Review by z-adams — APPROVED** [review]
Looks good to me. Make sure you squash all your commits into one before rebasing.

**Comment by EhWhoAmI** [comment]
Uh, I think your branch is a little messed up. Next time when you are working on a branch, don't pull master onto that branch. We can merge them. But this will be a bit of a mess to merge.

**Comment by z-adams** [comment]
What we should really do is just copy the file to a new branch and nuke this PR, start over from a clean branch.

**Comment by jonesmz** [comment]
@haennes

I'd really like to merge the changes you've got, but as the others pointed out, the branch has some conflicts.

Could you rebase your branch on master, so the only changes are your intended changes?

Or close this PR and start a new one, if you're more comfortable with that.

**Comment by jonesmz** [comment]
@haennes

Are you able to finish this work?

I will close this pull request on August 1st if we don't hear from you.

**Pull request closed without merging**
